spec/requests/api/v1/admin/event_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Event endpoint', type: :request do
+  let!(:admin) { create(:admin) }
+
+  let!(:user) { create(:user) }
+
+  let!(:event) { create(:event) }
+
+  let!(:event_to_update) { create(:event) }
+
+  let!(:auth_headers) { admin.create_new_auth_token }
+
+  let!(:invitation) { create(:invitation, user: user, event: event) }
+
+  let!(:event_to_create_with_user) do
+    {
+      'event' =>
+        {
+          'name' => 'evento_testing',
+          'address' => 'testing_address333',
+          'cost' => 200,
+          'start_time' => '2020-10-07T13:28:06.419Z',
+          'end_time' => '2020-10-07T15:28:06.419Z',
+          'cancelled' => false
+        },
+      'user_ids' => [user.id.to_i]
+    }
+  end
+
+  let!(:event_to_create_without_user) do
+    {
+      'event' =>
+        {
+          'name' => 'evento_testing',
+          'address' => 'testing_address',
+          'cost' => 200,
+          'start_time' => '2020-10-07T13:28:06.419Z',
+          'end_time' => '2020-10-07T15:28:06.419Z',
+          'cancelled' => false
+        }
+    }
+  end
+
+  let!(:event_data_update) do
+    {
+      'event' =>
+        {
+          'name' => 'evento_update',
+          'address' => 'evento_update',
+          'cost' => 500,
+          'start_time' => '2020-10-07T13:28:06.419Z',
+          'end_time' => '2020-10-07T15:28:06.419Z',
+          'cancelled' => false
+        },
+      'user_ids' => [user.id]
+    }
+  end
+
+  describe 'GET /api/v1/admin/events' do
+    context 'returns the two pre-existing events' do
```

Comment:
El context debería ser el contexto en el que se hacen las pruebas, por ejemplo en este caso debería ser `with multiple pre-existing events`, lo que ocurre por tener multiples eventos es lo que va en el `it` como bien pusiste en la línea de abajo.

Reply:
Sigue faltando un context sin eventos